airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/core/config/AirbyteConfiguredCatalog.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.core.config
```

Comment:
We're going to have to find a better name than `config` for this package, as "config" is already a dangerously-overloaded term.

Reply:
@postamar Agreed.  Not sure what to use, but definitely open to suggestions.

Reply:
The only thing my tired brain could come up with is "connector command inputs". That's two extra layers of package nesting, though. Perhaps that's OK.

Reply:
@postamar I am going to go with `command.option` as these are the command options from the command line.

Reply:
Nice! Thanks